packages/mc-scripts/src/commands/config:sync.js

```diff
@@ -2,6 +2,7 @@ const omit = require('lodash/omit');
 const prompts = require('prompts');
 const mri = require('mri');
 const chalk = require('chalk');
+const jsonDiff = require('json-diff');
```

Comment:
True. I agree with you. I think it's better to write a custom function to do the diffing since the data structure is known. 
I will update this. Thanks.

Reply:
Mainly we were looking for a library to show diff changes in a helpful way in the terminal. Computing the differences can also be done "manually" in this case yes.